src/ToDoList.tsx

```diff
@@ -26,7 +40,46 @@ export const ToDoList = () => {
                 </div>
             </div>
 
-            <button className='addButton'>+</button>
+            <button className='addButton' onClick={addNewTaskButton}>+</button>
+
+            <Modal
+                open={openWindow}
+                onClose={addNewTaskButton}
+                aria-labelledby="modal-modal-title"
+                aria-describedby="modal-modal-description"
+            >
+                <Box className='newTaskBox'>
+                    <Typography id='modal-modal-title' className='title' variant='h4'>
+                        Adding new task
+                    </Typography>
+                    <Typography id='modal-modal-description' className='content'>
+                        <div className='contentDiv'>
+                            <p>New task name</p>
```

Comment:
Tag `p` is not good choise here from semantic point of view
Use `label` instead